concepts/authorities.py

```diff
@@ -164,3 +165,16 @@ def get_by_namespace(namespace):
 
     return [ manager for manager in authority_managers
                 if manager.namespace == namespace ]
+
+
+def add(instance):
+    """
+    Adding the approved concept to Conceptpower
+
+    Parameters
+    -----------
+    instance: :class: '.Concept'
+    """
+
+    Conceptpower().create(settings.USER_ID, settings.PASSWORD, instance.label, instance.pos,
```

Comment:
What are these settings globals? They don't match anything in the settings modules.